lib/cmds/merge_cmds/export.ts

```diff
@@ -138,35 +136,24 @@ const exportEnvironmentMigration = async ({
     )
     await ensureDir(path.dirname(outputTarget))
   } catch (e) {
-    throw new Error('Something failed with the output file.')
+    throw new Error(
+      'Something failed with the output file. Ensure the path exists and is writable.'
+    )
   }
 
-  let migration: string | undefined
-  try {
-    migration = await callExportAppAction({
-      api: client,
-      appDefinitionId: mergeAppId,
-      createChangesetActionId: getAppActionId('create-changeset', host as Host),
-      exportActionId: getAppActionId('export-changeset', host as Host),
-      sourceEnvironmentId,
-      targetEnvironmentId,
-      spaceId: activeSpaceId
-    })
-
-    if (!migration) {
-      throw new Error(mergeErrors['MigrationCouldNotBeExported'])
-    }
```

Comment:
Is this something that can happen if no error is thrown above? In that case we still need this check

Reply:
We check if whats returned looks good here https://github.com/contentful/contentful-cli/blob/d4296c3e11e85ca12f481ed13a2861f37a9e1415/lib/utils/app-actions.ts#L100 so if migration doesn't exist at this point an error would have already been thrown and that makes this check redundant